src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -79,6 +80,15 @@ const SkillsQuizStepper = () => {
         });
       }
       setCurrentStep(STEP2);
+    } else if (goal === DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE && currentJob?.length > 0) {
+      if (currentJob?.length > 0) {
```

Comment:
This condition is already being checked in the above statement, we can discard it